<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleDocumentsReadingModeSelectorController</class>
 <widget class="QDialog" name="MultipleDocumentsReadingModeSelectorController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiple sequence reading mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="separateMode">
        <property name="text">
         <string>Separate sequence mode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mergeMode">
        <property name="text">
         <string>Merge sequence mode</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Number of 'unknown' symbols ('N 'for nucleic or 'X' for amino) between internal parts :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="internalGap">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Number of 'unknown' symbols ('N 'for nucleic or 'X' for amino) between files :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="fileGap">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>New document name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="newUrl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QCheckBox" name="saveBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save document</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QListWidget" name="listMergedDocuments">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="upperButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bottomButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MultipleDocumentsReadingModeSelectorController</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MultipleDocumentsReadingModeSelectorController</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>internalGap</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileGap</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>newUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MultipleDocumentsReadingModeSelectorController</receiver>
   <slot>sl_onChooseDirPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>listMergedDocuments</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>upperButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>bottomButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sl_onChooseDirPath()</slot>
  <slot>sl_onSaveNewDocument()</slot>
 </slots>
</ui>
